src/debugging/coreclr/LocalOSProvider.ts

```diff
@@ -2,6 +2,8 @@
  * Copyright (C) Microsoft Corporation. All rights reserved.
  *--------------------------------------------------------*/
 
+/* eslint-disable unicorn/filename-case */
```

Comment:
The rule does allow regex's in an ignore list, though it would probably take longer to invent a regex that properly excludes such uses without *over* excluding files than just to add the few inline exclusions where needed.